src/controllers/AuthLocalController.js

```diff
@@ -0,0 +1,56 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import sgMail from '@sendgrid/mail';
+import status from '../config/status';
+import { User } from '../queries';
+import * as helper from '../helpers';
+
+dotenv.config();
+
+/**
+ * [Users description]
+ */
+export default class AuthLocalController {
+  /**
+   * @param  {object} req
+   * @param  {object} res
+   * @return {object} return an object containing the confirmation message
+   */
+  static async sendEmail(req, res) {
+    const { email } = req.body;
+    const result = await User.findOne({ email }); // check if the email exist
+    if (Object.keys(result).length <= 0) {
+      return res.status(404).send({
+        message: 'email not found..'
+      });
+    }
+    sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+    const isSent = process.env.NODE_ENV === 'test' ? null : await sgMail.send(helper.sendgridMailTemplate.resetPassword(email, 'noreply@authersheaven.com'));
```

Comment:
I believe we agreed on a `sendEmail` helper that anyone can use

Reply:
changes applied